### PR TITLE
Adding Polish and changing READMY

### PR DESCRIPTION
**Pull request by BANSAFAn**
Look at the latest edits, there I posted 2 ideas for the launcher

### CODE REVIEW

File: README/README.zh.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <p align="center">
   <a href="https://xmcl.app" target="_blank">
-    <img alt="Logo" width="100" src="xmcl-electron-app/icons/dark@256x256.png">
+    <img alt="Logo" width="100" src="https://github.com/BANSAFAn/x-minecraft-launcher/blob/master/README/dark%40256x256.png">
   </a>
```
Comment (ci010) [inline README/README.zh.md:4]: This url should be changed, since you're merging. It should be like `README/dark@256x256.png`?

File: img.png
Comment (ci010) [inline img.png:1]: This is unused, please remove this

**Review by ci010 — CHANGES_REQUESTED** [review]

File: README/README.zh.md (hunk 1 above)
Comment (BANSAFAn) [reply README/README.zh.md:4]: The problem is that when you look at the translated images, the images are not the same, it’s simply not visible, although there is a link there, I honestly can’t figure out how to solve this

Comment (BANSAFAn) [reply README/README.zh.md:4]: Google translator translated it wrong.**


Images are not visible in translations  
Only in READMY.md and that's all, the rest are not visible

**Pull request merged**